pyrt/renderer/simplert.py

```diff
@@ -50,17 +50,21 @@ def _shade(self, scene: Scene, ray: Ray, hitrecord: HitRecord) -> tuple:
 
     def _shadow(self, scene: Scene, hitrecord: HitRecord) -> tuple:
         # is hitpoint in shadow ?
-        fs = 1.0
+        fs = 0.0
```

Comment:
What is fs? Why is it zero and why is it divided by 4 after cycle

Reply:
fs - total intensity at a point
In the absence of light sources is equal to 0
If the point is in the shadow of the object, then the intensity drops by 4 times (it was in the project)